lib/tds/types.ex

```diff
@@ -1585,13 +1588,14 @@ defmodule Tds.Types do
     date = :calendar.gregorian_days_to_date(@year_1900_days + days)
     hour = trunc(mins / 60)
     min = trunc(mins - hour * 60)
+
     {date, {hour, min, 0}}
     |> NaiveDateTime.from_erl!()
   end
 
   def encode_smalldatetime(nil), do: nil
 
-  def encode_smalldatetime(%{calendar: _}=naive_datetime) do
+  def encode_smalldatetime(%{calendar: _} = naive_datetime) do
```

Comment:
We can keep this, but if ecto mssql adapter is well covered with edge cases +  default loaders and dumpers handle case for naivedatetime, this should be sufficient except for those who use tds directly.